examples/vite-app-embed/src/App.tsx

```diff
@@ -37,9 +35,9 @@ function App() {
           setIsEmbedOpen(false);
         }}
         isEmbedOpen={isEmbedOpen}
-        hostname={String(
-          import.meta.env['VITE_PUBLIC_HOSTNAME'] ?? 'api.hume.ai',
-        )}
+        // hostname={String(
+        //   import.meta.env['VITE_PUBLIC_HOSTNAME'] ?? 'api.hume.ai',
+        // )}
```

Comment:
accidental commit?

Reply:
Yup, bringing this back. Thanks for catching!